crypto/evp/evp_lib.c

```diff
@@ -403,7 +403,7 @@ int EVP_CIPHER_CTX_iv_length(const EVP_CIPHER_CTX *ctx)
 
     params[0] = OSSL_PARAM_construct_size_t(OSSL_CIPHER_PARAM_IVLEN, &v);
     rv = evp_do_ciph_ctx_getparams(ctx->cipher, ctx->provctx, params);
-    if (rv == EVP_CTRL_RET_UNSUPPORTED)
+    if (rv == OSSL_RET_UNSUPPORTED)
```

Comment:
Here you are changing `EVP_CTRL_RET_UNSUPPORTED=-1` to `OSSL_RET_UNSUPPORTED=-2`? 

_(several locations)_